Scripts/AppRemoval/RemoveApps.ps1

```
@@ -34,129 +34,273 @@ function RemoveApps {
+                    if ($appData.AppId -is [array]) {
+                        foreach ($id in $appData.AppId) { $script:AppRemovalMethodCache[$id.Trim()] = $method }
+                    }
+                    else {
+                        $script:AppRemovalMethodCache[$appData.AppId.Trim()] = $method
+                    }
                 }
             }
         }
-        catch {
-            Write-Verbose "Something went wrong while trying to remove $($app): $_"
+        catch { }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)